### ISSUE TITLE
AirConditioner:VariableRefrigerantFlow with ThermostatOffsetPriority crashes with blank Zone Name for Master Thermostat Location

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5536]
## Issue overview

User file crashes [here](https://github.com/NREL/EnergyPlus/blob/ffaabe11cd573fa5493636d9926a177878396c3d/src/EnergyPlus/HVACVariableRefrigerantFlow.cc#L6474-L6476) with these inputs for thermostat control:

```
AirConditioner:VariableRefrigerantFlow,

  ,                                                        !- Zone Name for Master Thermostat Location
  ThermostatOffsetPriority,                                !- Master Thermostat Priority Control Type
```

It is expecting a zone name for the master thermostat location.  So, either the code needs to change here, or the getinput and docs needs to be changed.
### Details
- Develop as of today or so ffaabe11cd 
- Helpdesk ticket 11189
### Checklist
- [x] Defect file added to DefectFiles\5536
- [x] Ticket added to Pivotal for defect (development team task)
- [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@rraustad This should be straightforward to fix?

**Comment by rraustad** [comment]
That should be easily fixed [here](https://github.com/NREL/EnergyPlus/blob/ffaabe11cd573fa5493636d9926a177878396c3d/src/EnergyPlus/HVACVariableRefrigerantFlow.cc#L2460-L2470) which is used [here](https://github.com/NREL/EnergyPlus/blob/ffaabe11cd573fa5493636d9926a177878396c3d/src/EnergyPlus/HVACVariableRefrigerantFlow.cc#L3589) to set the MasterZoneTUIndex pointer.

**Comment by mjwitte** [comment]
@rraustad Looks like the error message for MasterThermostatPriority also needs some work.

**Comment by rraustad** [comment]
OK, just noticed that this code is in 2 places . Once for the old VRF model ((line 1823), and once for the new VRF model (line 2460). The new VRF model does not have an input field for Zone Name for Master Thermostat Location or Master Thermostat Priority Control Type and assumes a TStat in every zone and LoadPriority is the control so I won't touch the code there (maybe I'll just add the correct code in case in the future these different control options are allowed).

**Comment by rraustad** [comment]
New warnings for old VRF model from defect file:

```
** Severe  ** AirConditioner:VariableRefrigerantFlow = "VRF MULTISPLIT UNIT 1ST VRF HEAT PUMP"
**   ~~~   ** Zone Name for Master Thermostat Location must be entered when Master Thermostat Priority Control Type = THERMOSTATOFFSETPRIORITY
** Severe  ** AirConditioner:VariableRefrigerantFlow = "VRF MULTISPLIT UNIT 2ND VRF HEAT PUMP"
**   ~~~   ** Zone Name for Master Thermostat Location must be entered when Master Thermostat Priority Control Type = THERMOSTATOFFSETPRIORITY
** Severe  ** AirConditioner:VariableRefrigerantFlow = "VRF MULTISPLIT UNIT 3RD VRF HEAT PUMP"
**   ~~~   ** Zone Name for Master Thermostat Location must be entered when Master Thermostat Priority Control Type = THERMOSTATOFFSETPRIORITY
** Severe  ** AirConditioner:VariableRefrigerantFlow = "VRF MULTISPLIT UNIT 4TH VRF HEAT PUMP"
**   ~~~   ** Zone Name for Master Thermostat Location must be entered when Master Thermostat Priority Control Type = THERMOSTATOFFSETPRIORITY
** Severe  ** AirConditioner:VariableRefrigerantFlow = "VRF MULTISPLIT UNIT COMMON VRF HEAT PUMP"
**   ~~~   ** Zone Name for Master Thermostat Location must be entered when Master Thermostat Priority Control Type = THERMOSTATOFFSETPRIORITY
**  Fatal  ** GetVRFInput: Errors found in getting AirConditioner:VariableRefrigerantFlow system input. Preceding condition(s) causes termination.
```

**Comment by rraustad** [comment]
This isn't the correct fix. All zones are required to have a thermostat and there is no master zone.

**Comment by mjwitte** [comment]
Closed via #5537
